core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -312,6 +316,22 @@ public void initialise(final String graphId, final Schema schema, final StorePro
             }
         }
     }
+    /**
+     * Gets this Store's creation timestamp for {@link GetGraphCreatedTime} operation.
+     *
+     * @return the Store's creation timestamp.
+     */
+    public String getCreatedTime() {
+        return createdTime.toString();
+    }
+    /**
+     * Sets this Store's creation timestamp.
+     *
+     * @param createdTime the LocalDateTime to store as the graph creation date.
+     */
+    public void setCreatedTime(final LocalDateTime createdTime) {
```

Comment:
As above, just check if we want to be able to do this. `createdTime` feels like it should be immutable but I'm not sure